README.md

```diff
@@ -68,6 +68,11 @@ Table of contents
 
 # Project management / Agile development
 
+## eXtreme Programming (XP)
+
+ - [What is Extreme Programming (XP)?](https://www.agilealliance.org/glossary/xp/#:~:text=Extreme%20Programming%20(XP)%20is%20an,engineering%20practices%20for%20software%20development.) [agilealliance.org] → `What` `Why` `How`
```

Comment:
The `#:~:text=`  parameter in the URL is something that may be added by your search engine ([more info](https://wicg.github.io/scroll-to-text-fragment/)). The "real" url is `https://www.agilealliance.org/glossary/xp/`

May you edit your PR and remove the text fragment?

Other links are fine 👍

Reply:
No problem. I updated the link

Reply:
Perfect, thank you ;)